pkg/rulefmt/rulefmt.go

```diff
@@ -14,14 +14,15 @@
 package rulefmt
 
 import (
+	"bytes"
 	"context"
 	"io/ioutil"
 	"strings"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
-	yaml "gopkg.in/yaml.v3"
```

Comment:
Leave this as-is, it makes it more obvious what it is being imported as.

Reply:
Sure.